components/client/src/main/java/com/hotels/styx/client/netty/connectionpool/NettyConnectionFactory.java

```diff
@@ -160,7 +146,7 @@ public Builder name(String name) {
          * Sets number of client worker threads.
          *
          * @param clientWorkerThreadsCount number of client worker threads
-         * @return this builder
+         * @return this NettyConnectionFactory.Builder
```

Comment:
Are these changes strictly necessary (see 2 similar changes below)? They are just brief descriptions of return value and don't really need the full class name IMO.